src/site/content/en/blog/lighthouse-whats-new-6.0/index.md

```diff
@@ -0,0 +1,475 @@
+---
+title: "What's New in Lighthouse 6.0"
+authors:
+  - cjamcl
+date: 2020-05-18
+hero: lighthouse-logo.svg
+alt: Lighthouse logo.
+tags:
+  - blog # blog is a required tag for the article to show up in the blog.
+  - fast
+  - lighthouse
+---
+
+Today we're releasing Lighthouse 6.0!
+
+[Lighthouse](https://github.com/GoogleChrome/lighthouse/) is an automated website auditing tool that
+helps developers with opportunities and diagnostics to improve the user experience of their sites.
+It's available in Chrome DevTools, npm (as a Node module and a CLI), or as a browser extension (in
+[Chrome](https://chrome.google.com/webstore/detail/lighthouse/blipmdconlkpinefehnmjammfjpmpbjk) and
+[Firefox](https://addons.mozilla.org/en-US/firefox/addon/google-lighthouse/)). It powers many Google
+services, including [web.dev/measure](https://web.dev/measure/) and [PageSpeed
+Insights](https://developers.google.com/speed/pagespeed/insights/).
+
+Lighthouse 6.0 is available immediately on npm and in [Chrome
+Canary](https://www.google.com/chrome/canary/). Other Google services that leverage Lighthouse will
+get the update by the end of the month. It will land in Chrome Stable in Chrome 84 (mid-July).
+
+To try the Lighthouse Node CLI, use the following commands:  
+```bash
+npm install -g lighthouse
+lighthouse https://www.example.com --view
+```
+
+This version of Lighthouse comes with a large number of changes that are all listed in the
+changelog. (**TODO: link to changelog)**. We'll cover the highlights in this article.
```

Comment:
https://github.com/GoogleChrome/lighthouse/releases/tag/v6.0.0

`[listed in the 6.0 changelog]`  probably